### DISCUSSION TITLE
Synapses

### DISCUSSION THREAD

**Discussion by fcavarretta-ualr** [BlueBrain/BluePyOpt#382] [General]
I would like to use the BPO to determine the optimal configuration of synapses, with number of synapses and conductance peak as free parameters, while the objectives would relate to the somatic response an existing neuron model.

Does BPO include any Evaluator class that would enable this kind of optimization ?

**Comment by wvangeit** [comment]
Would this be in Neuron?
You might have to write your own parameters class like in here:
https://github.com/BlueBrain/BluePyOpt/blob/master/bluepyopt/ephys/parameters.py
For the synapse you can use this as an example:
https://github.com/BlueBrain/BluePyOpt/blob/master/examples/expsyn/ExpSyn.ipynb

**Reply by fcavarretta-ualr** [reply]
Yes, it would be in NEURON.
